testing/README.md

```diff
@@ -27,7 +27,7 @@ CLUSTER=kubeflow-testing
 NAMESPACE=kubeflow-test-infra
 ```
 
-After starting `kubectl proxy` you can connect to it at
+After starting `kubectl proxy` you can connect to it at 
```

Comment:
After starting `kubectl proxy` on `127.0.0.1:8001`, you can connect to the argo UI via the local proxy at: <url here>

Reply:
Updated accordingly. Please have a look at it.